content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -2,898 +2,998 @@
+        Impuestos Internos)` in the chatter.
+      - Post the invoice again.
+
+     .. image:: chile/rejected-invoice.png
+        :alt: Message when an invoice is rejected.
+        :align: center
 
 Crossed references
 ~~~~~~~~~~~~~~~~~~
 
-When the Invoice is created as a result of another fiscal document, the information related to the
-originator document must be registered in the Tab Cross Reference, which is commonly used for
-credit or debit notes, but in some cases can be used on Customer Invoices as well. In the case of
-the credit and debit notes, they are set automatically by Odoo:
+When the invoice is created as a result of another fiscal document, the information related to the
+originator document must be registered in the :guilabel:`Cross-Reference` tab, which is commonly
+used for credit or debit notes, but in some cases can be used on customer invoices as well. In the
+case of the credit and debit notes, they are set automatically by Odoo.
 
-.. image:: chile/Chile21.png
+.. image:: chile/cross-reference-tab-registration.png
+   :alt: Crossed referenced document(s).
    :align: center
-   :alt: Invoice tab with origin document number and data.
 
-Invoice PDF Report
+.. _chile/electronic-invoice-pdf-report:
+
+Invoice PDF report
 ~~~~~~~~~~~~~~~~~~
 
-Once the invoice is accepted and validated by the SII and the PDF is printed, it includes the
-fiscal elements that indicate that the document is fiscally valid:
+Once the invoice is accepted and validated by the :abbr:`SII (Servicio de Impuestos Internos)` and
+the PDF is printed, it includes the fiscal elements that indicate that the document is fiscally
+valid:
 
-.. image:: chile/Chile22.png
+.. image:: chile/sii-validation-elements.png
+   :alt: SII Validation fiscal elements.
    :align: center
-   :alt: Barcode and fiscal elements in the invoice report.
 
 .. important::
-   If you are hosted in Odoo SH or On-Premise, you should manually install the ``pdf417gen``
-   library. Use the following command to install it: ``pip install pdf417gen``.
+   If you are hosted in Odoo SH or On-Premise, you should manually install the `pdf417gen
+   <https://pypi.org/project/pdf417gen/>`_ library. Use the following command to install it:
+   :command:`pip install pdf417gen`.
 
-Commercial Validation
+Commercial validation
 ~~~~~~~~~~~~~~~~~~~~~
 
 Once the invoice has been sent to the customer:
 
-1. DTE partner status changes to “Sent”.
-2. The customer must send a reception confirmation email.
-3. Subsequently, if all the commercial terms and invoice data are correct, they will send the
-   Acceptance confirmation, otherwise they send a Claim.
-4. The field DTE acceptation status is updated automatically.
+#. :guilabel:`DTE Partner Status` changes to :guilabel:`Sent`.
+#. The customer must send a reception confirmation email.
+#. Subsequently, if all the commercial terms and invoice data are correct, they will send the
+   acceptance confirmation, otherwise they send a claim.
```

Comment:
Suggestions for brevity. Also, not really sure who "they" are in the original sentence. Might be useful to specify

```suggestion
#. Subsequently, if commercial terms and invoice data are correct, an acceptance confirmation is
   sent; otherwise, a claim is sent.
```